youtube/yt_data_extract/common.py

```diff
@@ -570,3 +570,12 @@ def extract_items(response, item_types=_item_types,
             ctoken = new_ctoken
 
     return items, ctoken
+
+def dbg_assert(val, msg):
+    import os
+    if val: return
+
+    if os.getenv('YT_EXTR_DBG'):
+        raise AssertionError(msg)
+    else:
+        print(f"W: {msg}")
```

Comment:
Remove, don't want to add an ad-hoc debugging framework just for logging unknown ctypes. Especially not one that uses os environment variables instead of the in-built settings. Can think about adding better debugging message handling in youtube-local in the future but not like this